M2/Macaulay2/packages/FastLinAlg.m2

```diff
@@ -1569,8 +1742,8 @@ doc ///
         orderType: Symbol
             a valid monomial order, such as {\tt GRevLex}
     Outputs
-        R1: ZZ
-           a polynomial ring
+        b: Boolean
+           true if 
```

Comment:
The thought here was left incomplete.

Reply:
Worse, the function doesn't return a Boolean, this is now fixed.